specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -3261,6 +3261,108 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/billingAccounts/default/lineOfCredit/default": {
+      "get": {
+        "tags": [
+          "LineOfCredits"
+        ],
+        "x-ms-examples": {
+          "LineOfCreditBySubscription": {
+            "$ref": "./examples/LineOfCreditBySubscription.json"
+          }
+        },
+        "operationId": "LineOfCredits_Get",
+        "description": "Get the current line of credit.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/LineOfCredit"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/billingAccounts/default/lineOfCredit/default/increase": {
+      "post": {
```

Comment:
Why are you modeling this as a post request? It would be more compliant to support put on lineOfCredit to change the value. Going with post will change default RBAC required to /action instead of /write and it won't work in template deployments either.

Reply:
Sure, Thanks! Updated to Put.

Reply:
While implementing, I thought about few more points:
1. Currently Get api has lots of properties which cannot be set during put. CreditLimit is the only property which is required to increase credit limit. Would it make more sense to change it to patch? Or I can have a PUT operation which is going to update only 1 property and rest oll properties will be ignored if user sets them.
2. If yes, This is a long running operation where the response will be 202. Most of the credit increase requests are goes to manual review which takes at lease couple of days and max 2 weeks. Would it make more sense to leave it as POST? Can patch return 202?

Reply:
I have changed it to PUT and updated the PR.